### PR TITLE
Profile changes and TestRunner package inclusion

### PR DESCRIPTION
**Pull request by ksahilCF**

### CODE REVIEW

**Comment by ksahilCF** [comment]
@tivv @rmstar profile changes along with report creation

File: pom.xml [hunk 1]
```diff
@@ -619,6 +619,9 @@
           <systemPropertyVariables>
             <java.io.tmpdir>${project.build.directory}</java.io.tmpdir>
           </systemPropertyVariables>
+          <excludes>
+            <exclude>**/TestRunner.java</exclude>
+          </excludes>
```
Comment (tivv) [inline pom.xml]: Any changes should go into profile. I recommend disabling SureFire in the profile altogether  by setting "skip" property of the surefire plugin in the profile. Anyway, with the profile enabled regular unit test directory (src/test) is not used

File: pom.xml [hunk 2]
```diff
@@ -800,6 +803,67 @@
...
       </properties>
+      <build>
+      <plugins>
+        <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.18.1</version>
+        <configuration>
+          <includes>
+            <include>TestRunner.java</include>
```
Comment (tivv) [inline pom.xml]: Why do you need to include TestRunner for surefire? I think you should disable surefire at all, only failsafe should run

File: pom.xml (hunk 1 above)
Comment (ksahilCF) [reply pom.xml]: removed exclusions

File: pom.xml (hunk 2 above)
Comment (ksahilCF) [reply pom.xml]: skiptest property used

**Review by tivv — APPROVED** [review]

**Pull request merged**